WcaOnRails/db/migrate/20190818102517_create_bookmarked_competitions.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateBookmarkedCompetitions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :bookmarked_competitions do |t|
+      t.string :competition_id, null: false
+      t.integer :user_id, null: false
+    end
+    add_index :bookmarked_competitions, :competition_id
```

Comment:
Perhaps it would make sense to have a unique index on `[:competition_id, :user_id]` pair?

Reply:
Done.